### PR TITLE
#128: Introduce changelog and update recent changes

### PR DESCRIPTION
**Pull request by ottlinger**
As discussed in https://github.com/zjedi/hugo-scroll/issues/128 a changelog might help to structure newly added features and bugfixes.

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -7,3 +7,5 @@ Mention new feature also in relevant `readme.md` section.
 
 ## [GitHub issues](https://github.com/zjedi/hugo-scroll/issues)
 Used for tracking more complex topics and features.
+
+   * https://github.com/zjedi/hugo-scroll/issues/126 Add version information to site's footer by setting the general configuration option `enableGitInfo=true`
```
Comment (zjedi) [inline contributing.md:11]: Can you start a dedicated `changelog.md`? I see contributing.md more like an abstract description of the process.

Comment (ottlinger) [reply contributing.md:11]: ok, will open a fresh PR with a changelog. (Just have to get my merges into master as I get confused on my fork ;))

**Comment by zjedi** [comment]
Can you name it `CHANGELOG.md`? I think that's most widely used naming

File: README.md [hunk 2]
```diff
@@ -51,9 +56,13 @@ Visit as at {{<extlink text="Instagram" href="https://www.instagram.com/yourInst
 ### Customizing CSS
 Add a `custom_head.html`-file to your `layouts/partials`-directory. In there you may add a `<style>`-tag _or_ you may add a `<link>`-tag referencing your own `custom.css` (in case you prefer to have a separate `.css`-file). Checkout the [`custom_head.html`](https://github.com/zjedi/hugo-scroll/blob/master/exampleSite/layouts/partials/custom_head.html)-file from the `exampleSite`-directory to get started and to find more detailed instructions.
 
+## Changelog
+
+Project updates can be found in our [changelog](./CHANGELOG.md)
```
Comment (zjedi) [inline README.md]: not worth a dedicated section imho, people can see the file in the root folder

File: contributing.md (hunk 1 above)
Comment (ottlinger) [reply contributing.md]: Added a changelog and integrated it - feel free to merge and enhance. Thanks. @zjedi

**Comment by ottlinger** [comment]
Fixes #128 if merged.

**Comment by ottlinger** [comment]
@zjedi I hope to have fixed all of your review remarks. Feel free to merge and we do see how future enhancements get logged in the newly created changelog :)

**Pull request merged**